ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -39,67 +52,109 @@ const ResultGroup = styled.div`
 `
 
 const render = (props: Props) => {
-  const { results, selectionInterface, className } = props
+  const { results } = props
   if (results.length === 0) {
-    return (
-      <ResultItemCollection className={className}>
-        <div className="result-item-collection-empty">No Results Found</div>
-      </ResultItemCollection>
-    )
+    return getNoResultsView(props)
   }
+  return getResultView(props)
+}
+
+const getNoResultsView = (props: Props) => {
+  const { className } = props
   return (
-    <ResultItemCollection
-      className={`${className} is-list has-list-highlighting`}
-    >
-      {results.map(result => {
-        if (result.duplicates) {
-          const amount = result.duplicates.length + 1
-          return (
-            <ResultGroup key={result.id}>
-              <div className="group-representation">{amount} duplicates</div>
-              <div className="group-results global-bracket is-left">
-                <ResultItemCollection className="is-list has-list-highlighting">
-                  <MarionetteRegionContainer
-                    view={childView}
-                    viewOptions={{
-                      selectionInterface,
-                      model: result,
-                    }}
-                    replaceElement
-                  />
-                  {result.duplicates.map((duplicate: any) => {
-                    return (
-                      <MarionetteRegionContainer
-                        key={duplicate.id}
-                        view={childView}
-                        viewOptions={{
-                          selectionInterface,
-                          model: duplicate,
-                        }}
-                        replaceElement
-                      />
-                    )
-                  })}
-                </ResultItemCollection>
-              </div>
-            </ResultGroup>
-          )
-        } else {
-          return (
-            <MarionetteRegionContainer
-              key={result.id}
-              view={childView}
-              viewOptions={{
-                selectionInterface,
-                model: result,
-              }}
-              replaceElement
-            />
-          )
-        }
-      })}
+    <ResultItemCollection className={className}>
+      <div className="result-item-collection-empty">No Results Found</div>
     </ResultItemCollection>
   )
 }
 
+const getResultView = (props: Props) => {
+  const { results, selectionInterface, className, solrQuery } = props
+  const showingResultsFor = createShowResultText(solrQuery)
+
+  return (
+    <SolrQueryDisplay className={className}>
```

Comment:
❓ Are we passing the `className` down because we want to do some sort of conditional rendering/styling?